src/base/utils/net.cpp

```diff
@@ -91,6 +92,37 @@ namespace Utils
             return subnet.first.toString() + '/' + QString::number(subnet.second);
         }
 
+        QHostAddress canonicalIPv6Addr(const QHostAddress &addr)
+        {
+            // Link-local IPv6 textual address always contains a scope id (or zone index)
```

Comment:
Does users really binds to link-local IP? Is there some real world use case of that?

Reply:
Either I use wrong terminology or one of us is confused. User may choose (for whatever reason) to listen to a specific IPv6 local address of their network interface. They may want for traffic to go through only through that specific one.